.github/workflows/tests.yml

```diff
@@ -21,13 +21,25 @@ jobs:
           sudo iptables -F FORWARD  || true
           sudo ip6tables -F FORWARD || true
 
+      - name: set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
       - name: Install dependencies
         run: |
           sudo snap refresh
           sudo snap set lxd daemon.group=adm
           sudo lxd init --auto
           sudo snap install snapcraft --classic
           snap list
+          # Python
+          sudo python -m pip install --upgrade pip
+          sudo pip install flake8 pep8-naming boto3 argparse
```

Comment:
The argparse module should be in stdlib since 2.7 / 3.2 -- do we need to support earlier Pythons, or is there any other reasons to install?

Reply:
ack

Reply:
I have removed the explicit install of argparse